app/views/spree/admin/variants/_autocomplete.js.erb

```diff
@@ -29,3 +29,43 @@
     </div>
   </div>
 </script>
+<script type='text/template' id='variant_autocomplete_stock_template'>
+  <fieldset>
+    <legend align="center"><%= Spree.t(:select_stock) %></legend>
+      <table class="stock-levels" data-hook="stock-levels">
```

Comment:
all data-hooks should be removed, right?

Reply:
Some of them are used as CSS tags or to trigger select2 dropdown, but I'll remove the unused ones